src/Ng/Jobs/Catalog2DnxJob.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Logging;
+using NuGet.Services.KeyVault;
+using NuGet.Services.Metadata.Catalog;
+using NuGet.Services.Metadata.Catalog.Dnx;
+using NuGet.Services.Metadata.Catalog.Persistence;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.Services.Configuration;
+
+namespace Ng.Jobs
+{
+    public class Catalog2DnxJob : LoopingNgJob
+    {
+        private CommitCollector _collector;
+        private ReadWriteCursor _front;
+        private ReadCursor _back;
+
+        public Catalog2DnxJob(ILoggerFactory loggerFactory)
+            : base(loggerFactory)
+        {
+        }
+
+        public override void PrintUsage()
+        {
+            Console.WriteLine("Usage: ng catalog2dnx "
```

Comment:
>Console.WriteLine [](start = 12, length = 17)

if the logger was initialized, we should log there (please verify the logger was setup to write to Console).